ansible/examples/slave_libvirt.yml

```
@@ -181,10 +190,17 @@
         state: directory
         owner: "{{ jenkins_user }}"
 
-    - name: install the vagrant-libvirt plugin
+    - name: install the vagrant-libvirt plugin (non-RHEL8)
       shell: vagrant plugin install vagrant-libvirt
       become: yes
       become_user: "{{ jenkins_user }}"
+      when: not (ansible_os_family == "RedHat" and ansible_distribution_major_version == "8")
+
+    - name: install the vagrant-libvirt plugin (RHEL8)
+      shell: CONFIGURE_ARGS="with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib64" vagrant plugin install vagrant-libvirt
```

I think you could write it this way instead :

```suggestion
      command: vagrant plugin install vagrant-libvirt
      environment:
        CONFIGURE_ARGS: 'with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib64'
```

---

Fixed